### DISCUSSION TITLE
How to make a QCard wider?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#8984] [General - Components / Directives / etc] [answered]
I would like to use a QCard in a QDialog but I cannot make it wider than a fixed width:

```
<template>
  <q-dialog
    (...)
  >
    <q-card style="width: 100vw; height: 100vh">
```

This makes the card high, but not wide:

![image](https://user-images.githubusercontent.com/1345886/115619296-fc628600-a2f3-11eb-9d36-1a28fd068865.png)

How can I increase the width?

**Comment by susansiow** [comment, marked as answer]
This might help - https://quasar.dev/vue-components/dialog#sizing

**Reply by wsw70** [reply]
Thank you! This was exactly this!

**Reply by susansiow** [reply]
Most welcome!
